.styleci.yml

```diff
@@ -3,13 +3,4 @@ preset: recommended
 linting: true
 
 finder:
-    exclude:
-        - "tests"
-    name:
-        - "*.php"
-    path:
-        - "src"
-    not-path:
-        - "tests"
-    depth:
-        - "< 3"
```

Comment:
yes, I saw that, but as I said, I need to get into all features in the doc and complete the configuration file. This was done quickly with later pending work.

By the way, will you provide integration for bitbucket too ?